gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,21 +1,5 @@
-#
```

Comment:
Why delete the header?

Reply:
Gradle updates those files automatically. It's autogenerated code.